buttons/button_base.h

```diff
@@ -30,27 +31,36 @@ class ButtonBase : public Looper,
         current_modifiers |= button_;
       }
       while (DebouncedRead()) {
```

Comment:
This loop seems to have been extended further than it used to be. Is that intentional?

Reply:
Not sure what you mean.